pkg/cache/cache.go

```diff
@@ -30,32 +32,34 @@ import (
 )
 
 type CacheResponse struct {
-	RoleARN  string
-	Audience string
+	RoleARN        string
+	Audience       string
+	UseRegionalSTS bool
 }
 
 type ServiceAccountCache interface {
 	Start()
-	Get(name, namespace string) (role, aud string)
+	Get(name, namespace string) (role, aud string, useRegionalSTS bool)
 }
 
 type serviceAccountCache struct {
-	mu               sync.RWMutex // guards cache
-	cache            map[string]*CacheResponse
-	store            cache.Store
-	controller       cache.Controller
-	clientset        kubernetes.Interface
-	annotationPrefix string
-	defaultAudience  string
+	mu                 sync.RWMutex // guards cache
+	cache              map[string]*CacheResponse
+	store              cache.Store
+	controller         cache.Controller
+	clientset          kubernetes.Interface
+	annotationPrefix   string
+	defaultAudience    string
+	defaultRegionalSts bool
```

Comment:
femto-nit, this field name should really be `defaultRegionalSTS`

Reply:
done